crates/ruff_python_parser/tests/snapshots/invalid_syntax@except_stmt_missing_exception.py.snap

```diff
@@ -0,0 +1,155 @@
+                                    ),
+                                ],
+                            },
+                        ),
+                    ],
+                    orelse: [],
+                    finalbody: [],
+                    is_star: false,
+                },
+            ),
+            Try(
+                StmtTry {
+                    range: 92..165,
+                    body: [
+                        Pass(
+                            StmtPass {
+                                range: 101..105,
+                            },
+                        ),
+                    ],
+                    handlers: [
+                        ExceptHandler(
+                            ExceptHandlerExceptHandler {
+                                range: 106..123,
+                                type_: None,
+                                name: None,
+                                body: [
+                                    Pass(
+                                        StmtPass {
+                                            range: 119..123,
+                                        },
+                                    ),
+                                ],
+                            },
+                        ),
+                        ExceptHandler(
+                            ExceptHandlerExceptHandler {
+                                range: 124..140,
+                                type_: None,
+                                name: None,
+                                body: [
+                                    Pass(
+                                        StmtPass {
+                                            range: 136..140,
+                                        },
+                                    ),
+                                ],
+                            },
+                        ),
+                        ExceptHandler(
+                            ExceptHandlerExceptHandler {
+                                range: 141..165,
+                                type_: None,
+                                name: Some(
+                                    Identifier {
+                                        id: "exc",
+                                        range: 152..155,
+                                    },
+                                ),
+                                body: [
+                                    Pass(
+                                        StmtPass {
+                                            range: 161..165,
+                                        },
+                                    ),
+                                ],
+                            },
+                        ),
+                    ],
+                    orelse: [],
+                    finalbody: [],
+                    is_star: true,
+                },
+            ),
+        ],
+    },
+)
+```
+## Errors
+
+  |
+1 | try:
+2 |     pass
+3 | except as exc:
```

Comment:
Nit: What happens if you have `except as:`

Reply:
Two error messages as expected. I'll add this case, thanks!

```
Errors:
-------
Syntax Error: Expected one or more exception types at byte range 21..23
  |
1 | try:
2 |     pass
3 | except as:
  |        ^^ Syntax Error: Expected one or more exception types
4 |     pass
  |


Syntax Error: Expected name after `as` at byte range 23..24
  |
1 | try:
2 |     pass
3 | except as:
  |          ^ Syntax Error: Expected name after `as`
4 |     pass
  |

```